inc/autoload/class-scrollable-adminmenu.php

```diff
@@ -0,0 +1,41 @@
+<?php
+add_action( 'init', array( 'Enqueue_scrollable_adminmenu_Style', 'init' ) );
```

Comment:
In this context I would see not a singleton, so maybe we get in a more testable version in the feature. Please change them like https://github.com/bueltge/wordpress-multisite-enhancements/blob/master/inc/autoload/class-add-ssl-identifier.php#L10 Thanks a lot! I hope you feel not misunderstanding, but its easier for me if you solve that and I merge it.

Reply:
i've it change. i'll hope you'r fine with this.

Reply:
+1 Thanks again.